### DISCUSSION TITLE
Closure option for parent record on relationManager?

### DISCUSSION THREAD

**Discussion by blite** [filamentphp/filament#4359] [Help] [answered]
does anyone know if there is a closure option for the parent record on a relationManager?  

a more specific example would be if you wanted to use the customer record in a closure for the address records of https://demo.filamentphp.com/shop/customers/3/edit?activeRelationManager=0

I feel like this probably already exists I'm just overlooking it.

**Comment by danharrin** [comment, marked as answer]
`$livewire->ownerRecord`

**Comment by blite** [comment]
this worked but I did have to use 
```
MyRelationManager $livewire
```
instead of 
```
Livewire $livewire
```

which was unexpected.

edit:

Livewire was importing as 
```
use Livewire\Livewire;
```

but using 
```
use Livewire\Component as Livewire;
```
as in the docs does work fine

**Reply by sergioraldana** [reply]
Same here, but I don't know if there is a better way to achieve this:

```
use Livewire\Component as Livewire;

                Select::make('semana_id')
                ->options(function (Livewire $livewire) {
                    return Semana::where('curso_disponible_id', $livewire->ownerRecord->id)->pluck('numero', 'id');
                })
                ->required()
```
